### PR TITLE
Emmachanges --------------Assigned UI redesigned

### PR DESCRIPTION
**Pull request by Hemazyn**
Login, Sign-Up, Reset-Password UI updated

### CODE REVIEW

File: src/components/Cards.jsx [hunk 1]
```diff
@@ -3,17 +3,11 @@ const imageUrl = `https://s3-alpha-sig.figma.com/img/699d/1183/8d9716e61a45f9e95
 export const EventCard = () => {
   return (
     <div className="flex flex-col w-[320px] h-[602px] justify-around items-center">
```
Comment (Leeyah-123) [inline src/components/Cards.jsx]: These hard coded heights and widths are still there

File: src/components/Success.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                              </div>
+                              <div className="flex flex-col gap-5 md:gap-10 items-center">
+                                   <div className="grid place-items-center bg-primary-dark w-[100px] h-[100px] rounded-full">
+                                        <svg width="23" height="18" viewBox="0 0 23 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                             <path d="M0.506348 10.2608L3.12487 7.64227L7.92116 12.4367L20.123 0.234863L22.7415 2.85338L7.91931 17.6737L0.506348 10.2608Z" fill="white" />
+                                        </svg>
+                                   </div>
+                                   <span className="font-Bayon font-normal text-base uppercase text-primary-default">congratulations! your password has been updated</span>
+                              </div>
+                              <button type="submit" className="w-full  bg-primary-light uppercase font-Bayon font-normal text-[16px] leading-[28.91px] rounded-[10px] py-2 md:py-4">Continue</button>
```
Comment (ezeaniiandrew) [inline src/components/Success.jsx]: When clicked. this button currently redirects the user to the Reset password page. It should take the user to the Login page instead.

File: src/pages/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                              <form noValidate className="flex flex-col gap-5 md:gap-10">
+                                   <div className="px-5 flex flex-col gap-5 md:gap-10">
+                                        {/* <div className="flex flex-col gap-10"> */}
+                                        <div className="flex flex-col font-Bayon">
+                                             <h2 className="text-base md:text-md text-start uppercase text-primary-default">login</h2>
+                                             <p className="text-slate uppercase font-Bayon font-normal text-sm md:text-base">doesn’t have an account? <Link to="/signup" className="text-primary-default">sign up</Link></p>
+                                        </div>
+                                        {/* inputs */}
+                                        <div className="w-full flex flex-col font-Bayon gap-5 md:gap-[37px]">
+                                             <input type="text" name="first_name" required autoComplete="none" placeholder="enter user name"
```
Comment (ezeaniiandrew) [inline src/pages/Login.jsx]: Make the input element a reusable component. Also, having both "user name" and "email field" for login is redundant. One is enough, preferably the email field.

File: src/pages/SignUp.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                                                  className="w-full flex text-primary-default text-sm md:text-base  uppercase rounded-[10px] border border-white outline-none  placeholder:text- p-[10px] placeholder:text-primary-default"
+                                                  onInvalid={(e) => e.target.setCustomValidity("Please input first name here",)}
+                                                  onInput={(e) => e.target.setCustomValidity("")} />
+                                             <input type="email" name="email_address" autoComplete="none" placeholder="enter email"
+                                                  className="w-full flex text-primary-default text-sm md:text-base  uppercase rounded-[10px] border border-white outline-none  placeholder:text- p-[10px] placeholder:text-primary-default"
+                                                  required />
+                                             {/* </div> */}
+                                             <div className="flex flex-col gap-3 md:gap-5">
+                                                  <input type="password" name="password" placeholder="enter at lease 6 symbols"
```
Comment (ezeaniiandrew) [inline src/pages/SignUp.jsx]: Fix the typo "least" not "lease". Also, create a another input field for password confirmation

**Review by ezeaniiandrew — CHANGES_REQUESTED** [review]
Reduce the margin between the contents and button on these screens on mobile:
Screenshot_20240424-091055
Screenshot_20240424-091033

File: src/components/Success.jsx (hunk 2 above)
Comment (Hemazyn) [reply src/components/Success.jsx]: I will adjust this when we merge.

File: src/pages/SignUp.jsx (hunk 4 above)
Comment (Hemazyn) [reply src/pages/SignUp.jsx]: fix

Comment (Hemazyn) [reply src/pages/SignUp.jsx]: fixed.

File: src/pages/Login.jsx (hunk 3 above)
Comment (Hemazyn) [reply src/pages/Login.jsx]: All issue fixed.

**Review by Leeyah-123 — CHANGES_REQUESTED** [review]
Change "Enter at least..." to "Enter Password". You can write guidelines for the password underneath it like "must be 6 characters long" and things of the sort

image

File: src/components/Cards.jsx (hunk 1 above)
Comment (Hemazyn) [reply src/components/Cards.jsx]: password placeholder text changed and validation text.

**Review by Leeyah-123 — APPROVED** [review]

File: src/components/Success.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import { Link } from 'react-router-dom';
+
+const Success = () => {
+  return (
+    <>
+      <div className="h-screen flex flex-row">
+        <div className="h-full w-full lg:w-[450px] flex flex-col justifyevenly">
```
Comment (bibhestee) [inline src/components/Success.jsx]: Typo here justify-evenly not justifyevenly

**Review by bibhestee — CHANGES_REQUESTED** [review]

File: src/components/Success.jsx (hunk 5 above)
Comment (Hemazyn) [reply src/components/Success.jsx]: I forgot to delete it.

**Review by bibhestee — APPROVED** [review]

**Review by Leeyah-123 — APPROVED** [review]

**Review by ezeaniiandrew — APPROVED** [review]

**Pull request merged**